src/devcenter/azext_devcenter/vendored_sdks/devcenter_dataplane/operations/_operations.py

```diff
@@ -8720,6 +8722,7 @@ def _delete_environment_initial(
             project_name=project_name,
             environment_name=environment_name,
             user_id=user_id,
+            force=force,
             api_version=self._config.api_version,
```

Comment:
Can you update this as well?

Reply:
Thank you! I updated that same line in the other _operations but not this one!